Feature button position

bnaolu opened the pull request.

bnaolu committed:
Added "Add to List" button

bnaolu committed:
Changed tag to button.

bnaolu committed:
Changed button position and fixed syntax.

Previously, the "add to list" button was displayed under the product tags.

New position (web and mobile):
- displayed under "view more images".

pglevy commented on _layouts/product.html:
```diff
@@ -18,12 +18,15 @@ <h1 class="margin-top-0">{{ page.title }}</h1>
           <div class="tablet:grid-col tablet:padding-right-1">
             <img src="{{ page.feature-image }}" alt="{{ page.feature-image-description }}">
             <button class="usa-button usa-button--unstyled margin-y-2">View more images</button>
+            <div class="margin-y-2">
```
We don't need the extra div here. You can apply the margin spacing class to the button element directly.

bnaolu replied:
When I remove the div, the button becomes in-line with the "view image" link. Is that okay? 
![Screen Shot 2021-05-14 at 12 51 01 PM](https://user-images.githubusercontent.com/52218695/118303280-441f9c00-b4b3-11eb-9ffb-967fde2cae7f.png)

pglevy commented on _layouts/product.html:
```diff
@@ -18,12 +18,15 @@ <h1 class="margin-top-0">{{ page.title }}</h1>
             <img src="{{ page.feature-image }}" alt="{{ page.feature-image-description }}">
             <button class="usa-button usa-button--unstyled margin-y-2">View more images</button>
+            <div class="margin-y-2">
+              <button class="usa-button  usa-button--hover">Add to List</button>
```
1. Remove usa-button--hover class. This is just for presenting documentation of UI. The hover state will occur naturally as people interact with it on a live page.
2. Let's stick with sentence case for button copy, so "Add to list".

pglevy reviewed (changes requested):
Looking good, @bnaolu! Just had a couple minor things to tweak. You can work off the same branch and publish your new commits there. It will flow into this same PR.

bnaolu committed:
Applied changes from PR review

Added 'display-block' to the button. Without it, the button will be in-line with "View more images". Not sure which we prefer.

pglevy commented:
Nice work, @bnaolu!
